javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,32 @@
+//For Peter
+const Pwide = 8;
+const Pdeep = 10;
+const Phigh = 10;
+let PvolumeInMeters;
+PvolumeInMeters = Pwide * Pdeep * Phigh;
```

Comment:
Try to avoid shorthand naming as it makes the code harder to read.
I'd call these constants something like:
`const peterWide`, `const peterDeep`, etc.